libs/types/src/investments.rs

```diff
@@ -239,3 +239,59 @@ pub struct ExecutedForeignCollect<Balance, Currency> {
 	///   pool currency)
 	pub amount_remaining: Balance,
 }
+
+/// A representation of an investment portfolio consisting of free, pending and
+/// claimable pool currency as well as tranche tokens.
+#[derive(Encode, Decode, Default, Clone, Eq, PartialEq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+
+pub struct InvestmentPortfolio<Balance> {
+	/// The unprocessed invest order amount in pool currency
+	pub pending_invest_currency: Balance,
+	/// The amount of tranche tokens which can be collected for an invest order
+	pub claimable_tranche_tokens: Balance,
+	/// The amount of tranche tokens which can be transferred
+	pub free_tranche_tokens: Balance,
+	/// The amount of tranche tokens which can not be used at all and get
+	/// slashed
```

Comment:
I think the `and get slashed` is confusing. Wouldn't it be `and could can get clashed`.

Reply:
Yeah the wording is suboptimal as it the _can_ is implicitly connected to _can not be used at all_ as well as _(can) get slashed_. I will improve this!

Reply:
[60e9add](https://github.com/centrifuge/centrifuge-chain/pull/1608/commits/60e9add23d13fb680cef7fc8f9522bac59e27765)